src/components/Reactions/ReactionSelector.tsx

```diff
@@ -183,7 +183,8 @@ const UnMemoizedReactionSelector = React.forwardRef(
             const count = reactionCounts && reactionCounts[reactionOption.id];
 
             return (
-              <li
```

Comment:
to avoid invalid element nesting (`ul > button`), you should keep `li` and add the `button`.